src/coreclr/tests/src/CLRTest.Execute.Bash.targets

```diff
@@ -284,6 +294,13 @@ fi
 
 $(BashLinkerTestCleanupCmds)
       ]]></BashCLRTestLaunchCmds>
+      <BashCLRTestLaunchCmds Condition="'$(CLRTestKind)' == 'BuildAndRun' And $(TargetArchitecture) == 'wasm' ">
+      <![CDATA[
+cd WasmApp
+./run-v8.sh $(MsBuildProjectName)
```

Comment:
My generated <code>run-v8.sh</code> looks like this:

<pre>
v8 --expose_wasm runtime.js -- --run WasmTestRunner.dll $*
</pre>

What is <code>v8</code> and where is it supposed to be coming from?

Reply:
v8 is the javascript engine. The easiest way to install it is with jsvu: https://github.com/GoogleChromeLabs/jsvu. v8 should already be present on the machines in that helix queue, because the wasm libraries are already using it.

Doc for testing the libraries may be relevenat: https://github.com/dotnet/runtime/blob/master/docs/workflow/testing/libraries/testing-wasm.md